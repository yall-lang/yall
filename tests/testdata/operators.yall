(+ 1 2)
(|> a b)
(<+> a b c)
[++ a]
