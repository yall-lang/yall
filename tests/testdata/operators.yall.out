[
    Expression {
        kind: Item,
        values: [
            Identifier(
                "+",
            ),
            Number(
                "1",
            ),
            Number(
                "2",
            ),
        ],
    },
    Expression {
        kind: Item,
        values: [
            Identifier(
                "|>",
            ),
            Identifier(
                "a",
            ),
            Identifier(
                "b",
            ),
        ],
    },
    Expression {
        kind: Item,
        values: [
            Identifier(
                "<+>",
            ),
            Identifier(
                "a",
            ),
            Identifier(
                "b",
            ),
            Identifier(
                "c",
            ),
        ],
    },
    Expression {
        kind: List,
        values: [
            Identifier(
                "++",
            ),
            Identifier(
                "a",
            ),
        ],
    },
]
